__tests__/api/api.test.js

```diff
@@ -34,15 +34,74 @@ describe("GET /api/topics", () => {
 });
 
 describe("GET /api", () => {
-  test("200: responds with a status of 200 and an array of endpoints on the body", () => {
+  test("200: responds with a status of 200 and an object describing all the available endpoints on the API", () => {
     return request(app)
       .get("/api")
       .expect(200)
       .then(({ body }) => {
         expect(body).toBeInstanceOf(Object);
-        expect(body).toHaveProperty("GET /api", expect.any(Object));
-        expect(body).toHaveProperty("GET /api/topics", expect.any(Object));
-        expect(body).toHaveProperty("GET /api/articles", expect.any(Object));
+        expect(body).toMatchObject({
+          "GET /api": {
+            description:
+              "serves up a json representation of all the available endpoints of the api",
+          },
+          "GET /api/topics": {
+            description: "serves an array of all topics",
+            queries: [],
+            exampleResponse: {
+              topics: [{ slug: "football", description: "Footie!" }],
+            },
+          },
+          "GET /api/articles": {
+            description: "serves an array of all articles",
+            queries: ["author", "topic", "sort_by", "order"],
+            exampleResponse: {
+              articles: [
+                {
+                  title: "Seafood substitutions are increasing",
+                  topic: "cooking",
+                  author: "weegembump",
+                  body: "Text from the article..",
+                  created_at: "2018-05-30T15:59:13.341Z",
+                  votes: 0,
+                  comment_count: 6,
+                },
+              ],
+            },
+          },
+        });
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: responds with a status of 200 and an object representing article_id 1", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Object);
+        const { article } = body;
+        expect(article).toMatchObject({
+          article_id: 1,
+          title: "Living in the shadow of a great man",
+          topic: "mitch",
+          author: "butter_bridge",
+          body: "I find this existence challenging",
+          created_at: "2020-07-09T20:11:00.000Z",
+          votes: 100,
+          article_img_url:
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+        });
+      });
+  });
+  test("404: responds with a status of 404 and a custom message of Article_id Not Found", () => {
+    return request(app)
+      .get("/api/articles/99")
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("Article_id Not Found");
```

Comment:
Please write another test (400), where the ID is invalid. For example '/api/articles/banana'